packages/server/src/models/agent-channel.js

```diff
@@ -0,0 +1,49 @@
+module.exports = (sequelize, DataTypes) => {
+  const AgentChannel = sequelize.define('AgentChannel', {
+    id: {
+      type: DataTypes.UUID(),
+      primaryKey: true,
+      defaultValue: sequelize.fn('uuid_generate_v4'),
+      allowNull: false,
+    },
+    agentId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    channelId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    primaryPointScore: {
+      type: DataTypes.FLOAT,
+      allowNull: false,
+    },
+    numberOfPredictions: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    numberOfQuestionsScored: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
```

Comment:
Is it important that ``numberOfPredictions`` and ``numberOfQuestionsScored`` are stored in the db? They seem to be caches. ``primaryPointScore`` too, though that is a bit more computation.

Reply:
If we do store them in the db, maybe we could add ``cache`` to the name to make it obvious what they are doing? Like, ``numberOfPredictionCache``

Reply:
Also, how do these get updated as they change? Is there logic for that yet? 

How about instead of doing it this way, you just calculate these on demand? I don't think we need to allow users to filter for these in the graphQL layer.

Reply:
1. Since we all are not sure about future code changes here we use "views" instead of "tables". "Views" are quite the same SQL queries, but they are already saved in the DB to make SQL queries right to them as well as to ordinary tables (for instance, "SELECT * FROM ViewA"). 
2. Hence, "views" are dynamic. Here no need to update anything. 
3. To get counters from DB is faster then make another DB queries.

Reply:
Oh nice, makes sense. I didn't notice that you were using Views.